### DISCUSSION TITLE
What is your favorite sport?

### DISCUSSION THREAD

**Discussion by swindlemek** [NAIST-SE/AttractingContributors#421] [General]
Please feel free to tell me your favorite sport.
Thank you.

https://github.com/swindlemek/FavoriteSport

**Comment by aru1702** [comment]
I do a little sport, but I prefer badminton, cycling, and tracking.
How about you?

See my repo: https://github.com/aru1702/AttractingContributors-TouhouFumo
Hop into my discussion: https://github.com/NAIST-SE/AttractingContributors/discussions/407
